lightning/src/ln/channelmanager.rs

```diff
@@ -2340,6 +2376,14 @@ impl<ChanSigner: ChannelKeys, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>
 		Ok(())
 	}
 
+	// Signal to the ChannelManager persister that there are updates necessitating persisting to disk.
+	fn persist_updates(&self) {
+		let &(ref persist_mtx, ref cnd) = &*self.persistence_lock;
+		let mut persistence_lock = persist_mtx.lock().unwrap();
+		*persistence_lock = true;
+		cnd.notify_all();
```

Comment:
trivial: `notify_once` is more intuitive here, but we don't actually expect multiple waiters, so it doesn't actually matter

Reply:
Hm, why is that more intuitive? Also I guess theoretically it could be better to be flexible in case a user has multiple persisters listening for updates?

Reply:
Note that `mpsc::channels` won't allow support for multiple persisters (unless aggregated behind a single persister) since it only support a single consumer.

Reply:
Oh, that seems like a bit of a dealbreaker to me then just because flexibility is nice